ROOT_Sistema_Inventario/Producto/templates/producto_crear_form.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+        <h1>HOLA QUE TAL</h1>
+        <h3>{{ form.as_table }}</h3>
+</body>
```

Comment:
Falta el boton para submitear el form
```suggestion
    <inpyt type="submit">Crear</input>
</body>
```